common-rest/src/test/java/org/uniprot/api/rest/validation/FakeIdsDownloadRequest.java

```diff
@@ -0,0 +1,43 @@
+package org.uniprot.api.rest.validation;
+
+import lombok.Builder;
+import lombok.Data;
+
+import org.uniprot.api.rest.request.IdsSearchRequest;
+
+/**
+ * @author sahmad
+ * @created 26/07/2021
+ */
+@Data
+@Builder
+public class FakeIdsDownloadRequest implements IdsSearchRequest {
```

Comment:
If you agree to change the validator name, make sure you also rename the fake class as well